<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonInfoDisplayKit</class>
 <widget class="QWidget" name="PersonInfoDisplayKit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">#widget
{
	background-color: rgb(255, 255, 255);
	font: 16pt &quot;微软雅黑&quot;;
	color: rgb(0, 0, 0);
	border:10px;
	border-color: rgb(209, 209, 209);
	border-style: solid;
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1,2,1,2,1,2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1,1,2,1,2,4,0,0">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_personname">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>杰瑞</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_personsex">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 13pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>男</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_personage">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 13pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>22岁</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_personbrithday">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 13pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>2000/8/23</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_Industry">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 13pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>IT</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_occupation">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 13pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>c++软件开发</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>438</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDelete">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 0, 0);
background-color: rgba(255, 255, 255,0);
font: 20pt &quot;微软雅黑&quot;;
border:none;</string>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="midLineWidth">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,2,3">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,9">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>tel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_tel">
            <property name="styleSheet">
             <string notr="true">font: 11pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>15575423419</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,9">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>QQ</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_qq">
            <property name="styleSheet">
             <string notr="true">font: 11pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>3234210243</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,9">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>email</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_email">
            <property name="styleSheet">
             <string notr="true">font: 11pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>19173489702@163.com</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="midLineWidth">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,20">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>住址：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_address">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>长沙市岳麓区保利麓谷林语G区16栋202室</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="midLineWidth">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,3,1">
        <property name="leftMargin">
         <number>15</number>
        </property>
        <property name="rightMargin">
         <number>15</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_recentupdate">
          <property name="styleSheet">
           <string notr="true">font: 14pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255,0);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="text">
           <string>最近更新：2023-1-16</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>778</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttondetail">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;
background-color: rgba(255, 255, 255,0);
color: rgb(85, 170, 0);
font: 16pt &quot;微软雅黑&quot;;</string>
          </property>
          <property name="text">
           <string>&lt;&lt;详情</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
